test/popd.js

```diff
@@ -114,3 +115,22 @@ test('Test that rootDir is not stored', t => {
   shell.popd(); // no more in the stack
   t.truthy(shell.error());
 });
+
+test('quiet mode', t => {
```

Comment:
Could we also test `stdout` for the case where `-q` is *not* passed? The test should have the same layout.

Reply:
Almost did this before, but wasn't sure if you'd want them. Done now.